code-the-dream-react/src/AddTodoForm.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+    function handleAddTodo(event) {
+        event.preventDefault();
+        let todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        document.getElementById("formValue").reset();
```

Comment:
another way you could have done it was to use the `event.target.reset()` instead `document.getElementById("formValue").reset();`

Reply:
That makes sense. Thank you!